libvirt/tests/src/virtual_disks/virtual_disks_nvme.py

```
@@ -9,105 +10,312 @@
+    """
+    if vm.is_dead():
+        vm.start()
+    session = vm.wait_for_login()
+    old_partitions = utils_disk.get_parts_list(session)
+    session.close()
+    vm.destroy(gracefully=False)
+
+    pci_addr_in_dict = get_usable_nvme_pci_address(test)
+
+    # Delete partitions if exist on nvme device
+    process.run("sfdisk --delete  /dev/nvme[0-9]n[0-9]",
```

Ruff (0.14.7)

Function call with `shell=True` parameter identified, security issue

(S604)